<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="msg">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>346</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="3">
      <widget class="QLabel" name="helpValidade">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLineEdit" name="linenumero">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="2">
      <widget class="QLineEdit" name="linedata">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="5" colspan="2">
      <widget class="QPushButton" name="buttonDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="labelCheckCPF">
       <property name="styleSheet">
        <string notr="true">color rgb(78, 154, 6)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Mostrar</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="labelCheckData">
       <property name="styleSheet">
        <string notr="true">color rgb(78, 154, 6)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelSenha">
       <property name="text">
        <string>Senha</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="labelCheckSenha">
       <property name="styleSheet">
        <string notr="true">color rgb(78, 154, 6)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="labelCodigo">
       <property name="text">
        <string>Código de Segurança</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLineEdit" name="linecodigo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QLineEdit" name="linesenha">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="helpCPF">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="helpSenha">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="labelCheckCodigo">
       <property name="styleSheet">
        <string notr="true">color rgb(78, 154, 6)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="helpCartao">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="helpCodigo">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Validade</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="labelCartao">
       <property name="text">
        <string>Cartão de Crédito</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="labelCheckCartao">
       <property name="styleSheet">
        <string notr="true">color rgb(78, 154, 6)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLineEdit" name="linecpf">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCPF">
       <property name="text">
        <string>CPF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
